pkg/quarterdeck/db/migrations/0006_projects.sql

```diff
@@ -0,0 +1,15 @@
+-- Add columns to the organization_projects table.
+
+BEGIN;
+
+-- Owner of the project.
+ALTER TABLE organization_projects ADD COLUMN owner_id BLOB NOT NULL DEFAULT '';
+
+UPDATE organization_projects
+SET owner_id = (
+    SELECT user_id FROM organization_users
+    WHERE organization_id = organization_projects.organization_id AND role_id = 1
+    ORDER BY created LIMIT 1)
+WHERE owner_id = '';
```

Comment:
Do we need a test for this migration or do you think deploying to staging is good enough?

Reply:
Deploying to staging is probably good enough.